pages/users/user_list.php

```diff
@@ -31,6 +42,13 @@
                 <td class="phone-number">
                     <?= $user->phone ?>
                 </td>
+                <?php if ($can_add_user): ?>
+                    <td class="">
+                        <a href="/licencies/<?= $user->id ?>/supprimer" class="destructive">
+                            <i class="fas fa-trash"></i>
```

Comment:
Very nice to have action buttons like this :) good idea!

Reply:
Actually depending on what you want to do, you can take inspiration from the dropdown in `event_view`